packages/stream-metadata/src/environment.ts

```diff
@@ -1,59 +1,33 @@
 import * as dotenv from 'dotenv'
+import { getWeb3Deployment } from '@river-build/web3'
 
-import { ChainConfig } from './types'
-import deploymentData from '@river-build/generated/config/deployments.json'
+import { Config } from './types'
 
 const isDev = process.env.NODE_ENV === 'development'
 const envFile = isDev ? '.env.localhost' : '.env'
 
-console.log('NODE_ENV:', process.env.NODE_ENV, 'isDev:', isDev, 'envFile:', envFile)
-
 dotenv.config({
 	path: envFile,
 })
 
 export const SERVER_PORT = parseInt(process.env.PORT ?? '443', 10)
-export const config = makeConfig(deploymentData, process.env.RIVER_ENV ?? 'omega')
-
-console.log('config:', config)
-
-interface DeploymentsJson {
-	[riverEnv: string]: {
-		river: {
-			chainId: number
-			addresses: {
-				riverRegistry: string
-			}
-		}
-	}
-}
-
-interface AllChainConfig {
-	[riverEnv: string]: {
-		chainId: number
-		riverRegistry: string
-	}
-}
-
-function makeConfig(deploymentsJson: DeploymentsJson, riverEnv: string): ChainConfig {
-	const allChainConfig: AllChainConfig = {}
-
-	for (const key in deploymentsJson) {
-		const envConfig = deploymentsJson[key]
-		if (envConfig.river) {
-			allChainConfig[key] = {
-				chainId: envConfig.river.chainId,
-				riverRegistry: envConfig.river.addresses.riverRegistry,
-			}
-		}
-	}
-
-	if (!allChainConfig[riverEnv].chainId || !allChainConfig[riverEnv].riverRegistry) {
-		throw new Error('chainId or riverRegistry undefined')
-	}
+export const config = makeConfig(process.env.RIVER_ENV, process.env.RIVER_CHAIN_RPC_URL)
```

Comment:
is this PR properly merged? I thought these changes went in already.

Reply:
not yet - this is a stacked PR. Awaiting parent PRs to merge first.

Reply:
Review in this order:

1. https://github.com/river-build/river/pull/727 - make env  & config deployment-ready
2. https://github.com/river-build/river/pull/729 - pino logger
3. https://github.com/river-build/river/pull/726 - eslint and lint fixes